pom.xml

```diff
@@ -1197,6 +1197,17 @@
                         <preparationGoals>clean verify -DskipTests</preparationGoals>
                     </configuration>
                 </plugin>
+
+                <plugin>
+                    <groupId>org.sonatype.plugins</groupId>
+                    <artifactId>nexus-staging-maven-plugin</artifactId>
+                    <version>1.6.8</version>
+                    <extensions>true</extensions>
```

Comment:
I just copied from the project's github project. My general understanding is that `extensions` generally introduce a new packaging for projects, which clearly is not why we are using this plugin. Let me understand this and remove this if not needed , otherwise comment why it is needed

Reply:
This is needed so that nexus-staging plugin can replace the instances of `maven-deploy-plugin` with itself.